src/menu.rs

```diff
@@ -10,6 +10,7 @@ pub struct Menu {
     text: CloneCell<String>,
     fg: Color,
     entries: Vec<Box<Entry>>,
+    activated: CopyCell<bool>,
```

Comment:
It's a type alias for `Cell`. @jackpot51  labeled it `CopyCell` to make it easier to differentiate which cell to use for which. `CloneCell` is a helper type for `RefCell`.